lib/genesis/helpers.go

```diff
@@ -16,8 +16,7 @@ import (
 	"github.com/ChainSafe/gossamer/dot/types"
 	"github.com/ChainSafe/gossamer/lib/common"
 	"github.com/ChainSafe/gossamer/lib/crypto"
-	"github.com/ChainSafe/gossamer/lib/runtime"
-	"github.com/ChainSafe/gossamer/lib/trie"
+	"github.com/ChainSafe/gossamer/lib/runtime/constants"
```

Comment:
New subpackage containing just a few hex trie keys, to avoid cycle dependency hell. Also an example why we need moare subpackages.

Reply:
@qdm12 the `constants` package you introduce here just contains some constants that is only used by production code while initializing the node and building the Genesis struct `dot/state/initialize.go`, I would suggest to move those constants to `lib/genesis` since they are specifically keys at the genesis file

Reply:
These are are used by both wasmer and genesis packages (and wasmer imports genesis). We can thus either:

- keep it as is in a separate dependency-free package
- duplicate the constants in both wasmer and genesis

But it feels a bit better to have them in that constants package, what do you think?

Reply:
Move to `lib/genesis/keys.go`

Reply:
Also these constants are used by some tests in the wasmer package, not just production code.